### PR TITLE
Add baseline indicators to upgrade-interactive entries

### PR DESCRIPTION
**Pull request by darthtrevino**
**What's the problem this PR addresses?**
This implements the upgrade-baselines feature proposed in #1980 , building on top of the ink upgrade in #2011 

**Checklist**
- [x] I have read the [Contributing Guide](https://yarnpkg.com/advanced/contributing).
- [x] I have set the packages that need to be released for my changes to be effective.
- [x] I will check that all automated PR checks pass before the PR gets reviewed.

### CODE REVIEW

**Comment by merceyz** [comment]
To keep the discussion could you instead reset https://github.com/yarnpkg/berry/pull/1980 to be equal to this PR and use that instead?

**Comment by darthtrevino** [comment]
@merceyz - Ah, I'm sorry - I was cleaning up the old PR and closed it before I saw your comment. GitHub isn't letting me re-open it.

**Comment by merceyz** [comment]
Alright, that's fine. It's not allowing you because you deleted the branch

File: packages/plugin-interactive-tools/sources/commands/upgrade-interactive.tsx [hunk 1]
```diff
@@ -197,15 +197,20 @@ export default class UpgradeInteractiveCommand extends BaseCommand {
       }, [
         descriptor.descriptorHash,
       ]);
+      const packageIdentifier = descriptor.scope ? `@${descriptor.scope}/${descriptor.name}` : descriptor.name;
```
Comment (paul-soporan) [inline packages/plugin-interactive-tools/sources/commands/upgrade-interactive.tsx:200]:
```suggestion
      const packageIdentifier = structUtils.stringifyIdent(descriptor);
```

**Comment by arcanis** [comment]
Perfect, thanks!

**Pull request merged**